mkdocs.yml

```diff
@@ -0,0 +1,106 @@
+site_name: lakefs-spec
+site_dir: public/docs
+site_url: https://lakefs-spec.github.io
+repo_url: https://github.com/appliedAI-Initiative/lakefs-spec
+edit_uri: -/edit/main/docs/
+
+# To validate all internal links exist. Does not work in ipynb files
+strict: false
+
+nav:
+  - Introduction: README.md
+  - Use Cases: use-cases.md
+  - User Guide: guides/
+  - API Reference: reference/
+  - CONTRIBUTING.md
+
+plugins:
+  - gen-files:
+      scripts:
+        - docs/_scripts/gen_api_ref_pages.py
+  - literate-nav:
+      nav_file: SUMMARY.md
+  - section-index
+  - mkdocstrings:
+      handlers:
+        python:
+          paths: [src]
+          options:
+            docstring_style: numpy
+            docstring_section_style: spacy
+            line_length: 88
```

Comment:
The line length for `black` and `ruff` is 100 in this project, configured in the `pyproject.toml`.

Reply:
Fixed in 969313a